Yoast/wordpress-seo
# Make force rewrite title integration

## Summary


This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* It's hard to test this pr. However with some hacking you can get it done:
* Add this to your code:
```php
add_action( 'wp_head', function() { 
   echo '<title>This is a title rendered before the Yoast meta.</title>';
}, 1 ); 

add_action( 'wp_footer', function() { 
   echo '<title>This is a title rendered after the Yoast meta.</title>';
}, -2 ); 
```
* Load your frontend and see multiple titles are rendered.
* Remove these lines: https://github.com/Yoast/wordpress-seo/pull/13786/files#diff-1e1f51cea761bb1b2910ae2990b44fadR67-R75 and verify both titles are removed. 

## UI changes

## Documentation

## Quality assurance


Fixes #
